### ISSUE TITLE
Can't use curl with this addon

### ISSUE THREAD

**Issue by bonanitech** [hassio-addons/addon-homebridge#11]
I'm trying to use the homebridge-cmdswitch2 plugin as suggested in https://community.home-assistant.io/t/homekit-location-to-trigger-automations-in-home-assistant-using-a-software-switch-requires-no-homekit-hardware/17235/8

Unfortunately the scripts don't work because `curl` can't be found:

```
[8/26/2017, 9:29:39 AM] [CMD switch] Initializing cmdSwitch2 platform...
[8/26/2017, 9:29:39 AM] Loading 0 accessories...
[8/26/2017, 9:29:39 AM] [CMD switch] Initializing platform accessory 'Mauricio is Home'...
[8/26/2017, 9:29:39 AM] [CMD switch] Failed to determine Mauricio is Home state.
[8/26/2017, 9:29:39 AM] [CMD switch] /config/homebridge/mbonani_status.sh: line 1: /usr/bin/curl: not found
```

But `curl` is there in the host:

```
root@hassio:~# which curl
/usr/bin/curl
```

And the script run normally in the shell:

```
core-ssh:/config/homebridge# ./mbonani_status.sh 
{"attributes": {"friendly_name": "Mauricio at home"}, "entity_id": "input_boolean.mbonani_home", "last_changed": "2017-08-26T11:57:19.829657+00:00", "last_updated": "2017-08-26T11:57:19.829657+00:00", "state": "on"}
```

Can you help me on that?

**Comment by quavoce** [comment]
I also have the same problem. It looks like the following needs to be added to the docker file please:

apt-get -qq -y install curl

**Comment by bonanitech** [comment]
I hope @frenck helps us on fixing that.

**Comment by frenck** [comment]
I agree on adding `curl`since it would be useful in many cases.

**Comment by frenck** [comment]
Updated milestone, will be part of a larger project, see: https://github.com/orgs/hassio-addons/projects/2

**Comment by bonanitech** [comment]
Broken link :-(

**Comment by frenck** [comment]
The links works on my machine.
Nevertheless, GitHub linked it just above your comment "New build system & base images".

**Comment by bonanitech** [comment]
![screenshot 2017-10-11 17 41 04](https://user-images.githubusercontent.com/29807944/31468357-6e6302d4-aeab-11e7-993c-4db121b63249.png)

**Comment by frenck** [comment]
@mauriciobonani Ok, I've learned something new today. Organisation projects are not visible to the public and GitHub currently offers no way to make them public.

Nevertheless, it's on the board ;)

**Comment by bonanitech** [comment]
Thanks for the feedback

Looking forward for the new version release

**Comment by frenck** [comment]
Will be automatically be fixed as soon as #28 is implemented/closed.

**Comment by frenck** [comment]
This issue is fixed in v2.0.0 of the add-on.
